tests/modules/segment_postprocess/test_segment_postprocess.py

```diff
@@ -5,17 +5,18 @@
 import argschema
 import json
 import pytest
-import ophys_etl.transforms.postprocess_rois as post_rois
+import ophys_etl.modules.postprocess_rois.__main__ as post_rois
```

Comment:
> aesthetic objection to importing from __main__.py

I agree with the aesthetic objection, but in discussion with Nick yesterday, he schooled me that this is _canonical_, and, it provides nice default functionality.
```
For a package, the same effect can be achieved by including a __main__.py module, the contents of which will be executed when the module is run with -m.
```
https://docs.python.org/3/library/__main__.html
That was enough to make me abandon my aesthetics, because it allowed for empty `__init__.py` files.

Reply:
> Can we give `suite2p_wrapper` a more informative name?

`suite2p_wrapper` lets us run Suite2P from the command line, or as an `ArgSchemaParser` class without having to deal with Suite2P (which has a different way of calling things from the command line). This wrapper puts some clothes on it to make it look like things we normally see.
Both segmentation and registration call this module.
One might say "but, then this is a utility!" which could be argued. But, it is a free-standing, fully functional module/entrypoint. In the interest of keeping structure flat, I left it here.

Reply:
> I'm not a huge fan of importing from __main__.py

I generally agree, this looks weird. This was a simple, low-impact change to migrate to the new structure.

Reply:
Your proposal is for `__main__.py` to be a shim. Nick also suggested this as one option.
Personally, I'd like to open up the entrypoint to the module (`__main__.py`) and read the flow of the processing, rather than open up the entry point, find out what the name of the class is that is called and what file it is in, and then go over to that file to figure out what's happening.
What I've been trying to do as I do this is make `__main__.py` just have the `ArgSchemaParser` class that describes the flow (in the `run()` method, as I do it). again, this was a low-impact change for the reorg, and I moved additional functions out to `utils.py`.
This approach is currently falling apart with `dff` as some monkeypatch business is not working nicely with that.
So, my goal for the reorg is to get everything roughly in its `src/ophys_etl/modules` location, and. further tweaks within any  (and/or all) modules would come later.

I think if we want to discuss as a group how `__main__.py` looks and works, and whether to use:
```
class MyClass(ArgSchemaParser):
    def run(self):
         <stuff>
```
as a canonical structure(my preference) (or something else), either in `__main__.py` or in some other location, we definitely can discuss.
But, for the moment, I am just trying to get a flat, relatively consistent folder structure with modules localized somewhay.